pkgs/development/python-modules/poppler-qt5/default.nix

```diff
@@ -1,35 +1,78 @@
-{ buildPythonPackage, fetchPypi, lib, sip, qtbase, pyqt5, poppler, pkg-config, fetchpatch
+{ stdenv
+, lib
+, buildPythonPackage
+, isPy3k
+, fetchPypi
+, pythonPackages
+, sip_5
+, qtbase
+, qmake
+, pyqt5
+, pyqt-builder
+, poppler
+, pkg-config
+, fetchpatch
 , substituteAll
 }:
 
 buildPythonPackage rec {
-  pname = "python-poppler-qt5";
-  version = "0.24.2";
+  pname = "poppler-qt5";
```

Comment:
https://github.com/frescobaldi/python-poppler-qt5/blob/0c50dd05fdab563a0c9decc2ed784712f37c822f/setup.py#L4

python-poppler-qt5 is the actual library name.

Reply:
Reverted... I had removed it because it looked weird with the double python prefix.